### DISCUSSION TITLE
Specific Times for Output Splitting Tool

### DISCUSSION THREAD

**Discussion by Kalyxchews** [enzo1982/freac#533] [Help & Support]
I just installed v1.1.7, very happy to see the output splitting tool implemented, but I was wondering if there's a way to input custom track durations for the output files instead of each track being exactly the same length? For context, there's no metadata on a specific CD rip that I've made, so I can't automatically split it using this tool, but if I could input the time signatures myself, this problem would be solved.

**Comment by Lee-Carre** [comment]
@Kalyxchews

> […] if there's a way to input custom track durations for the output files instead of each track being exactly the same length? […] if I could input the time signatures myself, this problem would be solved.

While I'm personally unfamiliar with using them, I've read that one can use CueSheets to achieve what you describe.

I have no idea on the details/specifics of doing so, though. I suggest searching other discussions/tickets for that keyword.
